### PR TITLE
Support multiple showroom instances

### PR DESCRIPTION
**Pull request by bierik**

### CODE REVIEW

File: ftw/showroom/js/src/ftw.showroom.js [hunk 1]
```diff
@@ -42,7 +43,7 @@ module.exports = function Showroom(items = [], options) {
         </div>
         <span class="ftw-showroom-title">{{item.title}}</span>
         <div class="ftw-showroom-right">
-          <a id="ftw-showroom-close" class="ftw-showroom-button"></a>
+          <a class="ftw-showroom-close" class="ftw-showroom-button"></a>
```
Comment (elioschmutz) [inline ftw/showroom/js/src/ftw.showroom.js]: class-attribute is defined twice

**Comment by bierik** [comment]
@elioschmutz updated

**Comment by elioschmutz** [comment]
Does not work properly...

screen1

**Comment by bierik** [comment]
@elioschmutz updated

**Pull request merged**